eng/service.proj

```diff
@@ -35,4 +35,13 @@
              SkipNonexistentTargets="true" />
   </Target>
 
+  <!--Run Aggregate Updates to Source -->
+  <Target Name="AggregateUpdatesToSource" AfterTargets="Build" Condition="'$(UpdateSource)' == 'true' or '$(AZURE_DEV_UPDATESOURCES)' == 'true'" >
+    <PropertyGroup>
+      <UpdateSnippetsScriptPath Condition=" '$(UpdateSnippetsScriptPath)'=='' ">$(MSBuildThisFileDirectory)scripts/Update-Snippets.ps1</UpdateSnippetsScriptPath>
+      <ExportAPIScriptPath Condition=" '$(ExportAPIScriptPath)'=='' ">$(MSBuildThisFileDirectory)scripts/Export-API.ps1</ExportAPIScriptPath>
+    </PropertyGroup>
+    <Exec ConsoleToMSBuild="true" Command="pwsh -NonInteractive -executionpolicy Unrestricted -File $(UpdateSnippetsScriptPath) -ServiceDirectory $(ServiceDirectory)"/>
```

Comment:
@pakrym @tg-msft are these the only scripts we want folks to run?

Reply:
Export-API and snippets are the main two, yes.

Reply:
We should add comments here and at https://github.com/Azure/azure-sdk-for-net/blob/master/eng/scripts/CodeChecks.ps1 to keep these lists in sync.

Reply:
Should we instead run CodeChecks.ps1 in a mode that doesn't fail if there are changes?

Reply:
It also runs code generation that's extremely slow and does some other stuff. We might add a flag to it though to only run these two things.

Reply:
OK I'd suggest adding a flag to codechecks.ps1 to run the subset we would run here. That makes things more consistent to what CI would do as well.